src/reduxTypes.js

```diff
@@ -251,6 +251,16 @@ export type VideoChatProvider = $ReadOnly<{| name: 'jitsi_meet', jitsiServerUrl:
  *   support.
  * @prop mandatoryTopics - Whether topics are required in stream messages
  *   (see https://zulip.com/help/require-topics)
+ * @prop messageContentDeleteLimitSeconds - Messages sent more than this
+ *   many seconds ago cannot be deleted with this organization's message
+ *   deletion policy; see
+ *     https://zulip.com/help/configure-message-editing-and-deletion.
+ *   A `null` value means no limit: messages can be deleted regardless of
+ *   how long ago they were sent. A `number` value will not be 0.
+ * @prop messageContentEditLimitSeconds - Messages sent more than this many
+ *   seconds ago cannot be edited with this organization's message edit
+ *   policy; see
+ *     https://zulip.com/help/configure-message-editing-and-deletion.
```

Comment:
So our pattern here has been to describe the semantics directly like this, but I'd like to shift that: for a property like these that corresponds to something from the server API, let's instead identify what in the API it corresponds to, and link to the docs for that. When there's a difference between this property and the corresponding bit of API, then we can highlight that difference.

(I think the existing pattern was set at a time when there wasn't much API docs to speak of, so the options were to write it ourselves or do without.)

So e.g.
```suggestion
 * @prop messageContentDeleteLimitSeconds - Corresponds to
 *   realm_message_content_delete_limit_seconds in initial data.
 *   We use the Zulip Server 5.0+ convention that `null` means no limit.
 * @prop messageContentEditLimitSeconds - Corresponds to
 *   realm_message_content_edit_limit_seconds in initial data.
```

and then above all the specific properties, a link to https://zulip.com/api/register-queue for docs on the initial data.